docs/b2b-edition/specs/api-v3/company/company.yaml

```diff
@@ -3028,6 +3028,7 @@ components:
     authToken:
       type: apiKey
       name: authToken
+      description: An API key is a token that you provide when making API calls. Include the token in a header parameter called `authToken`.
```

Comment:
This is required to authenticate with the API, so I think that should be stated.